Resources/views/SubscriptionWall/delete.html.twig

```diff
@@ -4,8 +4,10 @@
     <div class="content_columns_holder">
         <div class="content_column_inner">
             <div class="results">
```

Comment:
I don't think it's necessary to make a dedicated view just for conformation. If you really want users to confirm that they want to delete the wall, then I would suggest to use a javascript confirm popup.